src/timeMachine/actions/queries.ts

```diff
@@ -244,15 +250,54 @@ export const setQueryByHashID = (queryID: string, result: any): void => {
     })
 }
 
+export const timeMachineQueryErrorNotification = (
+  results: RunQueryResult[],
+  dispatch
+) => {
+  for (const result of results) {
+    if (result.type === 'UNKNOWN_ERROR') {
+      throw new Error(result.message)
+    }
+
+    if (result.type === 'RATE_LIMIT_ERROR') {
+      dispatch(notify(rateLimitReached(result.retryAfter)))
+
+      throw new Error(result.message)
+    }
+
+    if (result.didTruncate) {
+      dispatch(notify(resultTooLarge(result.bytesRead)))
+    }
+  }
+}
+
+export const runTimeMachineQuery = (
+  queryText: string,
+  state: AppState,
+  abortController: AbortController
+) => {
+  const allBuckets = getAll<Bucket>(state, ResourceType.Buckets)
+  const allVariables = getAllVariables(state)
+
+  event('executeQueries query', {}, {query: queryText})
```

Comment:
This is moved code ([from here](https://github.com/influxdata/ui/blob/93002cdad8692f470353c6355f284a66ae84e785/src/timeMachine/actions/queries.ts#L278)), and uses the current event naming. Please let me know if there are reasons to change/update.

Reply:
It's probably fine to leave as is. I'm not sure if we're going to move forward with the new convention or not, since that was something Kristina and Fabian were implementing.

For context, when I was building the app-metrics system, I had always assumed that we'd use something like `location.action.outcome` to name events, so in this case an event might be named `'timeMachine.query.execute'`, since a lot of places I've worked have a similar naming convention because it provides context to people analyzing the data. Alex didn't want to do that and insisted that we do it in a more sentence-style way because he preferred it. After Kristina joined, she wanted to use a similar system to the one I described. Alex didn't want to and just ignored that and kept doing them (and kept the QX team doing them) in his preferred way. So over the past few months before the layoff, Kristina was working with Fabian to try to get all our event names consistent, because Fabian preferred the system Kristina and I wanted to use becaue it was easier for him to work with, as I understand it.